### PR TITLE
feat: add dashboard template list

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] `Error / Warning / Lint / Type`

### Description

- Added dashboard template list constant. Use it at CreateDashboardPage.vue.
- Added description to DashboardConfig interface.
- Added monthlyCostSummary template as a sample.

### Things to Talk About

Previous PR must be merged first.

### CODE REVIEW

File: src/services/dashboards/default-dashboard/helper.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const getDashboardLayoutWidgetInfoList = (widgetList: string[]): DashboardLayoutWidgetInfo[] => widgetList.map((widgetId) => {
+    try {
+        const widgetConfig = getWidgetConfig(widgetId);
+        const widgetInfo: DashboardLayoutWidgetInfo = {
+            widget_name: widgetConfig.widget_config_id,
+            title: widgetConfig.title ?? widgetConfig.widget_config_id,
+            widget_options: widgetConfig.widget_options ?? {},
+            size: widgetConfig.sizes[0],
+            version: '1',
```
Comment (yuda110) [inline src/services/dashboards/default-dashboard/helper.ts:13]: question: Do we have to specify `version` when request real API?

**Review by yuda110 — APPROVED** [review]
LGTM!!
Thanks for clear codes.

File: src/services/dashboards/config.ts [hunk 2]
```diff
@@ -18,3 +18,60 @@ export const GRANULARITY = {
...
+    start?: string;
+    end?: string;
+}
+
+type DashboardVariables = Record<string, any|any[]>;
+
+interface DashboardSettings {
+    date_range: {
+        enabled: boolean;
+    } & DateRange;
```
Comment (minseolee) [inline src/services/dashboards/config.ts:41]: 👀

**Review by minseolee — APPROVED** [review]
LGTM!!

File: src/services/dashboards/default-dashboard/helper.ts (hunk 1 above)
Comment (WANZARGEN) [reply src/services/dashboards/default-dashboard/helper.ts:13]: Hm.. I don't think so.. I think we can omit version in each widget info, and just specify to dashboard info. What do you think about it?

**Pull request merged**